src/content/scripting/playlist_parser_script.cc

```diff
@@ -157,11 +163,7 @@ void PlaylistParserScript::processPlaylistObject(const std::shared_ptr<CdsObject
     currentLine = new char[ONE_TEXTLINE_BYTES];
     currentLine[0] = '\0';
 
-#ifdef __linux__
-    currentHandle = std::fopen(obj->getLocation().c_str(), "re");
-#else
-    currentHandle = std::fopen(obj->getLocation().c_str(), "r");
-#endif
+    currentHandle = std::fopen(obj->getLocation().c_str(), linux ? "re" : "r");
```

Comment:
Wouldn't it make more sense to define/constexpr some `openMode = "re"` or `openMode = "r"` and use that in fopen instead of `linux ?:`? This way we can reuse if for other fopens

Reply:
not really. there aren't too many similar ones.